packages/frontend-2/components/project/page/settings/collaborators/Collaborators.vue

```diff
@@ -125,11 +156,20 @@ const onCollaboratorRoleChange = async (
   if (collaborator.inviteId) return
 
   loading.value = true
-  await updateRole({
-    projectId: projectId.value,
-    userId: collaborator.id,
-    role: newRole
-  })
+  if (isWorkspacesEnabled.value && project.value?.workspaceId) {
```

Comment:
I'd abstract this away under a new composable, in case there are multiple places where project roles can be changed, so that future devs don't have to think about which mutation should be invoked